ios/app/brave_core_main.mm

```diff
@@ -0,0 +1,71 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/ios/app/brave_core_main.h"
+
+#include "base/strings/sys_string_conversions.h"
+#include "brave/ios/app/brave_main_delegate.h"
+#import "brave/ios/browser/brave_web_client.h"
+#import "ios/chrome/app/startup_tasks.h"
+#include "ios/chrome/browser/application_context.h"
+#include "ios/chrome/browser/browser_state/chrome_browser_state.h"
+#include "ios/chrome/browser/browser_state/chrome_browser_state_manager.h"
+#include "ios/chrome/app/startup/provider_registration.h"
+#include "ios/public/provider/chrome/browser/chrome_browser_provider.h"
+#include "ios/web/public/init/web_main.h"
+
+@interface BraveCoreMain()
+{
+  std::unique_ptr<BraveWebClient> _webClient;
+  std::unique_ptr<BraveMainDelegate> _delegate;
+  std::unique_ptr<web::WebMain> _webMain;
+  ChromeBrowserState* _mainBrowserState;
+}
+@end
+
+@implementation BraveCoreMain
+
+- (instancetype)init {
+  if ((self = [super init])) {
```

Comment:
[This logic](https://github.com/brave/brave-core/pull/6213/files#diff-0d8ff2ee6ebd53c44ee99cf515f96740R167-R172) was added to ensure a CommandLine existed for the encryption PR, and probably needs to be removed now

Reply:
https://github.com/brave/brave-core/pull/6722/files#diff-0ea36e952461437e3c0166eb905d60b1472efac3d4609dd445b4ab15478383aaL174